src/promnesia/sources/hypothesis.py

```diff
@@ -1,12 +1,13 @@
 '''
 Uses HPI [[https://github.com/karlicoss/HPI/blob/master/doc/MODULES.org#myhypothesis][hypothesis]] module
 '''
-from ..common import Visit, Results, logger, Loc
+from ..common import Loc, Results, Visit, join_tags, logger
 
 
 def index() -> Results:
-    from . import hpi
     import my.hypothesis as hyp
+
+    from . import hpi
```

Comment:
nope, `from . import hpi` has to be first, otherwise `import my.hypothesis` fails with obscure error if HPI isn't installed

Reply:
Thanks!.  Fixed in 429d882.

Reply:
But shouldn't these import-error manifest themselves?

Reply:
Yep, in that sense it's redundant -- it just gives a bit friendlier error message for people less familiar with python